fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -509,6 +520,12 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = QUERY_TIMEOUT)
     private int queryTimeoutS = 300;
 
+    @VariableMgr.VarAttr(name = IS_SET_USE_PAGE_CACHE)
```

Comment:
just one session variable is sufficient,  keep usePageCache, its default value should be true.

Reply:
> just one session variable is sufficient, keep usePageCache, its default value should be true.

updated